docs/topic_guide/channel_response.rst

```diff
@@ -0,0 +1,148 @@
+
+Pixel Solid Angle
+-----------------
+
+The pixel area, :math:`p`, is the angular area in the plane of the sky subtended by a single detector pixel.
+It has a physical type of solid angle per pixel.
+The units of the pixel area are typically expressed as :math:`\mathrm{sr}\,\mathrm{pixel}^{-1}`.
+Typically, this quantity can be determined as the product the spatial plate scale in each direction.
+In the FITS standard, these keys are denoted by "CDELTi", with "i" typically taking on values of 1 or 2.
+Note that the pixel area is sometimes confusingly referred to as the plate scale.
+However, here we explicitly define the plate scale to be the angular *distance* subtended by one side of a pixel.
+
+Effective Area
+--------------
+
+The effective area describes the instrument sensitivity as a function of wavelength and time.
+It is given by,
+
+.. math::
+
+    A_{\mathrm{eff}}(\lambda,t) = A_{\mathrm{geo}}M(\lambda)F(\lambda)Q(\lambda)D(\lambda,t)\quad[\mathrm{cm}^2]
+
+The effective area has a physical type of area.
+:math:`A_\mathrm{eff}(\lambda,t=0)` is defined as the effective area at the start of the mission.
+Each component of the effective area is described in detail below.
+
+Geometrical Area
+****************
+
+The geometrical collecting area, :math:`A_\mathrm{geo}`, is the cross-sectional area of the telescope aperture.
+It has a physical type of area.
+The units of the geometrical collecting area are commonly expressed as :math:`\mathrm{cm}^2`.
+For example, for a telescope with a circular aperture of diameter :math:`d`, the geometrical collecting area is :math:`A_\mathrm{geo}=\pi d^2/4`.
+
+Mirror Reflectance
+******************
+
+The mirror reflectance, :math:`M(\lambda)`, is a dimensionless quantity describing the efficiency of the mirror(s) in the instrument as a function of wavelength.
+If the instrument contains multiple mirrors (e.g. a primary and secondary mirror), this quantity is the product of the reflectance of each mirror.
+:math:`M(\lambda)` should always be between 0 and 1.
+
+Filter Transmittance
+********************
+
+The filter transmittance, :math:`F(\lambda)`, is a dimensionless quantity describing the efficiency of the filter(s) as a function of wavelength.
+This is typically calculated by computing the transmittance through a given compound of a specified thickness.
+In the case of a multilayer coating, the transmittance is the product of the transmittance of each layer of the coating.
+Similarly, if an instrument contains multiple filters (e.g. an entrance and focal-plane filter), this quantity is the product of the transmittance of each mirror.
+:math:`F(\lambda)` should always be between 0 and 1.
+
+Quantum Efficiency
+******************
+
+The quantum efficiency, :math:`Q(\lambda)`, is a dimensionless quantity describing the efficiency of the detector.
+:math:`Q(\lambda)` should always be between 0 and 1.
```

Comment:
There's some disagreement in the literature, but the quantum efficiency is usually the number of electron-hole pairs measured per photon. The quantity you're describing here is sometimes called the `effective` quantum efficiency.

Reply:
And wikipedia calls this quantity the [external quantum efficiency](https://en.wikipedia.org/wiki/Quantum_efficiency#Types)

Reply:
I'm perfectly happy to rename this "effective quantum efficiency" and link to that wikipedia entry.